vnc.sh

```diff
@@ -9,6 +9,8 @@ fi
 
 vncserver -kill $display
 vncserver -geometry 1750x1250 $display > /dev/null
-vncviewer localhost$display > /dev/null &
+if command -v vncviewer >/dev/null 2>&1; then
```

Comment:
You should expose port 9042 or whatever it is, so people can attach a desktop `vncviewer` to the container and watch the magic happen.